components/bl-mention-component/src/index.js

```diff
@@ -0,0 +1,124 @@
+import { useEffect, useState } from 'react';
+
+import primereact from './lib/core';
+
+const { Mention } = primereact.mention;
+const { cn } = BackendlessUI.CSSUtils;
+
+const DEFAULT_FIELD_BLACK_LIST = ['created', '___class', 'ownerId', 'updated', 'objectId'];
+
+export default function MentionComponent({ component, eventHandlers }) {
+  const {
+    trigger, suggestions, field, scrollHeight, autoHighlight, placeholder, delay, autoresize, rows, cols,
+    hideField, classList, style, display,
+  } = component;
+  const { onChange, onFocus, onBlur, onShow, onHide } = eventHandlers;
+
+  const [processedSuggestions, setProcessedSuggestions] = useState([]);
+  const triggers = useTriggers(trigger);
+  const [fieldBlackList, setFieldBlackList] = useState(DEFAULT_FIELD_BLACK_LIST);
+
+  useEffect(() => {
+    if (hideField?.length) {
+      const hideValues = hideField.split(',').map(el => el.trim());
+      setFieldBlackList(prev => [...prev, ...hideValues]);
```

Comment:
If for first render giddenFields = ['1'] , and for second  ['2']. Why we merge them?